### PR TITLE
Add a highlight to the active link in the side nav

### PR DESCRIPTION
**Pull request by el-mapache**
* The sidenav width is super tiny and breaking the line on the GitHub
Branches link.

**HANDY GIF**:
highlight-link-gif

Let me know your thoughts, this was like 5 minutes of work so its not a big deal if its not what we want to go with

### CODE REVIEW

**Comment by ericronne** [comment]
🤚 may i design? 😬

**Comment by el-mapache** [comment]
yeah design it up!

**Comment by ericronne** [comment]
_onnit!_

**Comment by ericronne** [comment]
Here's one way it could work …
image

Could the nav column could be widened to avoid line breaking (and ensure that there's enough space to the right of each nav item)?

File: frontend/sass/_side-nav.scss [hunk 1]
```diff
@@ -13,7 +13,20 @@
     &:hover svg g {
       fill: $link-color-active;
     }
+
+    &.side-nav-active-item {
+      background-color: #2d476a;
+      color: white !important;
+      padding: 5px !important;
```
Comment (line47) [inline frontend/sass/_side-nav.scss]: Is there a way this can be done without using `!important` ?

**Comment by ericronne** [comment]
@line47 do you mean adopting [this overall style](https://designsystem.digital.gov/components/) to the dashboard? That could be a simple solution.

Regardless, i think the overall design of the dashboard could benefit from a ground-up rethink (rather than piecemeal changes). I wish i had time to address it. :/

**Comment by line47** [comment]
@ericronne not necessarily I was just referring to the left nav in that example link has it's own width and formatting outside of the usa-grid system (which has baked in breakpoints where the grid shifts).

This would give a bit more flexibility for styling the left nav and not having it conform to a grid system.

File: frontend/sass/_side-nav.scss (hunk 1 above)
Comment (el-mapache) [reply frontend/sass/_side-nav.scss]: Probably, it will just involve a bunch of specificity selectors to override USWDS behavior

**Comment by el-mapache** [comment]
Closing for now as this is going to require a bit more planning to figure out how we want to present the side-nav going forward.

**Pull request closed without merging**